### ISSUE TITLE
Changes required to train the CGAN model

### ISSUE THREAD

**Issue by dikshantsagar** [mattragoza/LiGAN#66]
Hello, I am trying to train a CGAN model. Please confirm what changes need to be made either in the config file or in the code to get this working correctly. Currently, on the train.config file, I have changed the `model_type` to **CGAN** and under `disc_model`, changed the parameters of the model architecture to match the `generator`, changed the loss weights as follows `gan_loss` to 1.0 `steric_loss` to 1.0 and rest 0.0, then finally match the `disc_optim` learning rate to match the `gen_optim`
